charts/modules/infra/aws-eks-rbac/Chart.yaml

```diff
@@ -5,7 +5,7 @@ apiVersion: v2
 appVersion: "0.41.0"
 description: A Helm chart for IAC resources needed to access EKS cluster
 name: aws-eks-rbac
-version: "0.41.0"
+version: "0.41.0-1"
```

Comment:
0.47 is latest

Reply:
Yes, but underlying crossplane-aws-iam is still 0.41.0.